config/default.config.yml

```diff
@@ -104,3 +104,10 @@ database:
     user: ""
     password: ""
     name: ""
+
+# separate block db for storing block bodies (no archive beacon node required)
+blockDb:
+  engine: "none" # pebble / none
```

Comment:
Should also add `s3` to the comment and also have some commented out s3 config example?